src/app/project-detail/project-detail.page.html

```diff
@@ -46,7 +46,7 @@
           {{project.status | camelToTitle}}
         </ion-col>
       </ion-row>
-      <div class="custom-filed" *ngIf="project?.learningResources">
+      <div class="custom-filed" *ngIf="project?.learningResources.length">
```

Comment:
project?.learningResources?.length

Add like this

Reply:
modified